src/SQLiteDatabase.js

```diff
@@ -62,7 +66,7 @@ function escapeBlob(data) {
 
 function unescapeForIOSAndAndroid(rows) {
```

Comment:
Oh, doesn't it have to unescape as well for macOS?

Reply:
ah.. I'm not 100% sure, right now I haven't found any issues, but it probably does, do you know what do I have to do to run this function?

Reply:
I created #84, I will continue testing on my application but seems to be fine for now